autoload/test/ruby/cucumber.vim

```diff
@@ -4,7 +4,7 @@ endif
 
 function! test#ruby#cucumber#test_file(file) abort
   if a:file =~# g:test#ruby#cucumber#file_pattern
-    return empty(glob('features/**/*.php'))
+    return empty(glob('features/**/*.php')) && empty(glob('features/**/*.js'))
```

Comment:
I think we actually *do* want to activate CucumberJS when there are JS files in the `features/` directory, so I think the condition should be negated: `&& !empty(glob('features/**/*.js'))`

I was thinking that, just like Behat (the PHP version of Cucumber) is activated when there are PHP files in the `features/` directory, we could activate the Ruby Cucumber only if there are Ruby files in the `features/` directory, and CucumberJS only if there are JS files. What do you think?

Reply:
I stuck with the approach that was there because I'm not clear how vim-test deals with scenarios where multiple runners qualify. For example, how would vim-test address having _both_ `.rb` and `.php` files under the `features` directory? Does a capability currently exist to help address this conflict?

Reply:
For JavaScript we can check in package.json, is that sufficient or do we need something similar to check composer.json for PHP or the gem lock file for Ruby?

Reply:
> For example, how would vim-test address having both .rb and .php files under the features directory?

I'm just hoping that no setup will use two languages for implementing Cucumber steps 😃. I think we can ignore this possibility for now, and revisit it if someone opens a ticket.

Reply:
Okay folks, I've updated the algorithm for selecting which cucumber runner to execute.

Reply:
I'm scared of breaking existing Ruby users. The existing setup essentially has Ruby as the "default" runner for `.feature` files. I considered making the Ruby version search for cucumber in the `Gemfile` but thought that checking for `.rb` files in the `features` directory would be safer.